.github/workflows/basic-checks.yaml

```diff
@@ -8,6 +8,7 @@ jobs:
   Static-Analysis:
     strategy:
       matrix:
+        # note: bumping the go value should be done for all workflows. Make certain to check conditionals as well.
```

Comment:
What does "go value" refer to here? The version?

Reply:
yeah, I'll update to make more clear